python/dgl/mock_sparse2/sparse_matrix.py

```diff
@@ -273,12 +271,12 @@ def create_from_csr(
                  values=tensor([1., 1., 1., 1., 1.]),
                  shape=(3, 3), nnz=5)
     >>> # Specify shape
-    >>> A = create_from_csr(indptr, indices, shape=(5, 3))
+    >>> A = create_from_csr(indptr, indices, shape=(3, 5))
     >>> print(A)
     SparseMatrix(indices=tensor([[0, 1, 2, 2, 2],
```

Comment:
The indices here are still COO indices, right?

Reply:
Yes.